ods_ci/tests/Resources/Common.robot

```diff
@@ -155,7 +155,7 @@ Get RHODS Version
             ${RHODS_VERSION}=  Run  oc get csv -n ${OPERATOR_NAMESPACE} | grep "opendatahub" | awk -F ' {2,}' '{print $3}'
         END
     END
-    Log  Product:${PRODUCT} Version:${RHODS_VERSION}
+    Log To Console    Product:${PRODUCT} Version:${RHODS_VERSION}
```

Comment:
you can do both things by adding "console=yes" in the `Log` parameters

Reply:
Log to console doesn't log in the HTML report no? while Log with console=yes does both

Reply:
Good to know :) 
I'll update it.
Thank you!!